<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_file_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
	border-radius: 8px;
	border: 2px solid black;
}</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_path_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="model_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>641</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_x_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotate_x_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_x_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:8px;
background-color: white;
border: 2px solid black
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="move_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Movement</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scale_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Scaling</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="norm_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Normalization</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="min_norm_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="max_norm_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_norm_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_norm_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>630</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_y_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_z_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotate_y_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="rotate_z_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="move_z_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_y_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_z_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="move_y_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_x_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="move_x_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="move_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:8px;
background-color: white;
border: 2px solid black
}</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scale_x_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scale_x_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scale_y_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scale_z_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="scale_y_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="scale_z_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:8px;
background-color: white;
border: 2px solid black
}</string>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="norm_step_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
border-radius:8px;
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="norm_step_edit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>560</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-bottom: 2px solid rgb(92, 157, 255);
border-top: 2px solid rgb(92, 157, 255);
border-right: 2px solid rgb(92, 157, 255);
border-left: 2px solid rgb(92, 157, 255);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_y_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:8px;
background-color: white;
border: 2px solid black
}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_z_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir Next</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:8px;
background-color: white;
border: 2px solid black
}</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
